code/game/machinery/dna_infuser/organ_sets/rat_organs.dm

```diff
@@ -0,0 +1,130 @@
+	flash_protect = FLASH_PROTECTION_HYPER_SENSITIVE
+	eye_color_left = "#000000"
+	eye_color_right = "#000000"
+
+/obj/item/organ/internal/eyes/night_vision/rat/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/noticable_organ, "has deep, shifty black pupils, surrounded by a sickening yellow sclera.", BODY_ZONE_HEAD)
+	AddElement(/datum/element/organ_set_bonus, /datum/status_effect/organ_set_bonus/rat)
+
+
+
+///increases hunger, disgust recovers quicker, expands what is defined as "food"
+/obj/item/organ/internal/stomach/rat
+	name = "mutated rat-stomach"
+	desc = "Rat DNA infused into what was once a normal stomach."
+	disgust_metabolism = 3
+
+/obj/item/organ/internal/stomach/rat/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/noticable_organ, "salviates excessively.", BODY_ZONE_HEAD)
+	AddElement(/datum/element/organ_set_bonus, /datum/status_effect/organ_set_bonus/rat)
+
+/obj/item/organ/internal/stomach/rat/on_life(delta_time, times_fired)
+	. = ..()
+	owner.adjust_nutrition(-9 * HUNGER_FACTOR) //Hunger depletes at 10x the normal speed
+
+/obj/item/organ/internal/stomach/rat/Insert(mob/living/carbon/reciever, special, drop_if_replaced)
+	. = ..()
+	if(!ishuman(reciever))
+		return
+	var/mob/living/carbon/human/human_holder = reciever
+	var/datum/species/species = human_holder.dna.species
+	//mmm, cheese. doesn't especially like anything else
+	species.liked_food = DAIRY
+	//but a rat can eat anything without issue
+	species.disliked_food = NONE
+	species.toxic_food = NONE
+	RegisterSignal(human_holder, COMSIG_SPECIES_GAIN, PROC_REF(on_species_gain))
+
+/obj/item/organ/internal/stomach/rat/proc/on_species_gain(datum/source, datum/species/new_species, datum/species/old_species)
+	SIGNAL_HANDLER
+	new_species.liked_food = DAIRY
+	new_species.disliked_food = NONE
+	new_species.toxic_food = NONE
+
+/obj/item/organ/internal/stomach/rat/Remove(mob/living/carbon/stomach_owner, special)
+	. = ..()
+	if(!ishuman(stomach_owner))
+		return
+	var/datum/species/species = stomach_owner.dna.species
+	species.liked_food = initial(species.liked_food)
+	species.disliked_food = initial(species.disliked_food)
+	species.toxic_food = initial(species.toxic_food)
+
+	UnregisterSignal(stomach_owner, COMSIG_SPECIES_GAIN)
+
+
+
+/// makes you smaller, walk over tables, and take double damage
+/obj/item/organ/internal/heart/rat
+	name = "mutated rat-heart"
+	desc = "Rat DNA infused into what was once a normal heart."
+
+/obj/item/organ/internal/heart/rat/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/noticable_organ, "hunches over unnaturally!")
+	AddElement(/datum/element/organ_set_bonus, /datum/status_effect/organ_set_bonus/rat)
+
+/obj/item/organ/internal/heart/rat/Insert(mob/living/carbon/reciever, special, drop_if_replaced)
+	. = ..()
+	if(!ishuman(reciever))
+		return
+	var/mob/living/carbon/human/human_reciever = reciever
+	human_reciever.dna.add_mutation(/datum/mutation/human/dwarfism)
```

Comment:
subtype with a different name, 0 instability and locked set to TRUE?

Reply:
Why?